tests/unit/client/serial/test_rtu.py

```diff
@@ -1,12 +1,17 @@
 import pytest
 from serial import serial_for_url
 
-from umodbus.client.serial.rtu import send_message
+from umodbus.client.serial.rtu import send_message, read_coils
 
 
 def test_send_message_with_timeout():
-    """ Test if TimoutError is raised when serial port doesn't receive data."""
+    """ Test if TimoutError is raised when serial port doesn't receive enough
+    data.
+    """
     s = serial_for_url('loop://', timeout=0)
+    # as we are using a loop, we need the request will be read back as response
```

Comment:
Can you capitalize the first letter of the sentence and end the sentence with a dot?

Reply:
Sure, thing. If there is anything else that hits your eye, let me know. Open to any requested changes/